Userland/Libraries/LibWeb/DOM/EventDispatcher.cpp

```diff
@@ -181,13 +181,13 @@ void EventDispatcher::invoke(Event::PathEntry& struct_, Event& event, Event::Pha
         // 2. If event’s type attribute value is a match for any of the strings in the first column in the following table,
         //    set event’s type attribute value to the string in the second column on the same row as the matching string, and return otherwise.
         if (event.type() == "animationend")
-            event.set_type("webkitAnimationEnd"sv);
+            event.set_type(HTML::EventNames::webkitanimationend);
         else if (event.type() == "animationiteration")
-            event.set_type("webkitAnimationIteration"sv);
+            event.set_type(HTML::EventNames::webkitanimationiteration);
         else if (event.type() == "animationstart")
-            event.set_type("webkitAnimationStart"sv);
+            event.set_type(HTML::EventNames::webkitanimationstart);
         else if (event.type() == "transitionend")
-            event.set_type("webkitTransitionEnd"sv);
+            event.set_type(HTML::EventNames::webkittransitionend);
```

Comment:
Hmm, don't we lose the weird casing here? i.e `webkitAnimationEnd` vs `webkitanimationend`.

Reply:
Oh yeah, I thought it didn't matter. But now I see they are legacy event types and that's probably why they have the weird casing.

Reply:
Fixed